packages/nuxt/src/app/components/nuxt-island.ts

```diff
@@ -174,7 +174,7 @@ export default defineComponent({
     } else if (import.meta.server || !nuxtApp.isHydrating || !nuxtApp.payload.serverRendered) {
       await fetchComponent()
     }
-
+    console.log(html.value, error.value)
```

Comment:
```suggestion
```

Reply:
just saw it and pushed 😅 you're fast 🔥  !